src/Components/Notifications/Notification.tsx

```diff
@@ -1,6 +1,21 @@
-import { Flex, Text } from "@artsy/palette"
+import {
+  Flex,
+  Skeleton,
+  SkeletonBox,
+  SkeletonText,
+  Spacer,
+  Text,
+} from "@artsy/palette"
 import { createFragmentContainer, graphql } from "react-relay"
 import { Notification_me$data } from "__generated__/Notification_me.graphql"
+import { SystemQueryRenderer } from "System/Relay/SystemQueryRenderer"
+import { NotificationQuery } from "__generated__/NotificationQuery.graphql"
+import { useNotificationsContext } from "Components/Notifications/useNotificationsContext"
+
+export const SUPPORTED_NOTIFICATION_TYPES = [
+  "ARTWORK_ALERT",
+  "ARTWORK_PUBLISHED",
+]
```

Comment:
Right now, we only have designs for these notification types. I'll clarify whether we want to show a preview for other "not-yet" supported types or simply link to the notification's `targetHref`.